res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- TODO(Tomasito): Change this to the version in which KeyboardController support will be added-->
+<MixxxKeyboardPreset schemaVersion="1" mixxxVersion="2.0.0+">
+  <info>
+    <name>Keyboard Default</name>
+    <author>Mixxx</author>
+    <description>Default keyboard mapping. (Layout: en_US)</description>
+    <wiki>https://www.mixxx.org/manual/latest/chapters/appendix.html#appendix-keyboard</wiki>
+  </info>
+
+  <controller>
+    <group name="[AutoDJ]">
```

Comment:
I'm just thinking about something. keyboards send at lower level events like keyPressed, keyReleased, like midi controllers in fact. Then the system and its drivers and libraries interpret that to "letter x has been typed". What event type does the underlying Qt functions you use gives ? It could be interresting to have status (pressed, released) and being able to have different functions for press and release (like enable something on press, disable it on release). With that, you will also be able to map on dead keys (SHIFT, CTRL, etc...).